packages/atlas/src/api/queries/notifications.graphql

```diff
@@ -106,6 +106,29 @@ query GetNftNotifications($memberId: ID!, $limit: Int!) {
       title
     }
   }
+
+  commentCreatedEvents(
+    where: { OR: [{ videoChannel: { id_eq: $channelId } }, { parentCommentAuthor: { id_eq: $memberId } }] }
```

Comment:
Won't the first filter also catch comments that are replies made under your video (not replies to your own comments)?

Reply:
You're right, fixed.